core/src/main/java/org/apache/calcite/adapter/enumerable/EnumerableWindow.java

```diff
@@ -210,6 +210,9 @@ public Result implement(EnumerableRelImplementor implementor, Prefer pref) {
       List<AggregateCall> aggregateCalls = group.getAggregateCalls(this);
       for (int aggIdx = 0; aggIdx < aggregateCalls.size(); aggIdx++) {
         AggregateCall call = aggregateCalls.get(aggIdx);
+        if (call.ignoreNulls()) {
+          throw new UnsupportedOperationException("Do not support ignore nulls now");
+        }
         aggs.add(new AggImpState(aggIdx, call, true));
```

Comment:
It throws exception because I don't implement `IGNORE NULLS` for now. I want to make sure I am in the right way before implementing it.